### DISCUSSION TITLE
variable for "Invoice Text" field for projects

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3737] [Questions & Answers] [answered]
I just noticed that there is a field "invoice text" which can be filled in when creating a project:
<img width="900" alt="screenshot" src="https://user-images.githubusercontent.com/51003649/211169774-46c158fe-286d-4f00-87e8-973ba1e0181e.png">
From its name, I concluded that its content can be shown on an invoice - however, I was not able to find an according variable under [https://www.kimai.org/documentation/invoices.html#project-variables](url) 
Help is appreciated

**Comment by kevinpapst** [comment, marked as answer]
There is no special variable, but this field can overwrite the description.
If you group invoice entries by project, the entered text here will replace the invoice line description. 
Same goes for activity and customer.

**Comment by BeckeBauer** [comment]
I see, thank you for the explanation

**Comment by derdeimos** [comment]
Hi

I can use ${customer.invoice_text} in the invoice template and get the text I entered in this field under customer.
![image](https://github.com/kimai/kimai/assets/32060272/30d459f7-71aa-4ba1-a710-dd99bfb0cf21)

But how can I get the text from the field "Invoice Text" from the project?
![image](https://github.com/kimai/kimai/assets/32060272/ab5ab9fd-b71d-4aaa-875f-fc6fc42a2f8a)

To create an invoice I filter for one project.

**Reply by kevinpapst** [reply]
Currently missing as invoice variables. Next release will support `${project.invoice_text}` and `${activity.invoice_text}`

**Reply by derdeimos** [reply]
Ok, thank you
